nativescript-angular/zone-js/dist/zone-nativescript.js

```diff
@@ -1,12 +1,12 @@
 var Zone$1 = (function (global) {
     if (global.Zone) {
-        throw new Error('Zone already loaded.');
```

Comment:
Let's exclude the zone.js ' -> " transform since it's generated from the Angular code.